ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml

```diff
@@ -79,6 +79,19 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-aws-etcd-encryption
   timeout: 5h0m0s
+- always_run: false
+  as: netpol-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-netpol-v2
```

Comment:
please use netpol instead of netpol-v2
Below are the agreed default parameters for 24 node testing, so please add them here in the env
PODS_PER_NAMESPACE: 10
NETPOL_PER_NAMESPACE: 20
LOCAL_PODS: 10
SINGLE_PORTS: 5
PORT_RANGES: 5
REMOTE_NAMESPACES: 5
REMOTE_PODS: 5
CIDR: 5

Reply:
Thanks Anil, updated the defaults!